src/css/_footer.scss

```
@@ -35,11 +35,27 @@
   height: 28px;
 }
 
-@media screen and (max-width: 825px) {
-  .site-footer {
+@media only screen and (max-width: 767px),
+only screen and (min-width: 768px) and (max-width: 1023px) {
```

Can't this be simplified too?
```suggestion
@media only screen and (max-width: 1023px) {
```

---

yes